scripts/deployment/liquidity-mining/update-lm.py

```diff
@@ -106,32 +106,31 @@ def updateLMConfig():
     # SOV/rBTC - 25k SOV
     ALLOCATION_POINT_BTC_SOV = 25000 # (WR)BTC/SOV
 
-    # xUSD/rBTC - 25k SOV
-    ALLOCATION_POINT_BTC_XUSD = 25000 # (WR)BTC/XUSD
+    # xUSD/rBTC - 1 SOV
+    ALLOCATION_POINT_BTC_XUSD = 1 # (WR)BTC/XUSD
 
-    ALLOCATION_POINT_I_XUSD =  1 # iXUSD
+    # DLLR/rBTC - 25k SOV
+    ALLOCATION_POINT_BTC_DLLR =  25000 # (WR)BTC/DLLR
 
     ALLOCATION_POINT_DEFAULT = 1 # 12 tokens with 1 alloc point to account: (WR)BTC/USDT1 | (WR)BTC/USDT2 | (WR)BTC/DOC1 | (WR)BTC/DOC2 | (WR)BTC/BPRO1 | (WR)BTC/BPRO2 | (WR)BTC/MOC | (WR)BTC/FISH | (WR)BTC/RIF | (WR)BTC/MYNT | (WR)BTC/BNB | (WR)BTC/ETH
     ALLOCATION_POINT_CONFIG_TOKEN = MAX_ALLOCATION_POINT - ALLOCATION_POINT_BTC_SOV - ALLOCATION_POINT_BTC_XUSD \
-                                     - ALLOCATION_POINT_I_XUSD - ALLOCATION_POINT_DEFAULT * 12
+                                     - ALLOCATION_POINT_BTC_DLLR  - (ALLOCATION_POINT_DEFAULT * 12)
```

Comment:
i think it should be ALLOCATION_POINT_DEFAULT * 13 , because of iXUSD (add iXUSD then to the commented list above)

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/474/commits/e6e49d3004cecd0f4f30268d42c460819f22817e)